src/client/components/todos/todolist.jsx

```diff
@@ -9,7 +9,7 @@ export default React.createClass({
 
   propTypes: {
     // Whenever component prop is an immutable structure, use PureRenderMixin.
-    todos: React.PropTypes.instanceOf(immutable.List)
+    todos: React.PropTypes.instanceOf(immutable.Iterable)
```

Comment:
Hmm, I think that's bad idea. PropTypes should be strict. If someone will change List To Map, then React component can fail because Map doesn't have all methods of List.